src/routes/uniswap_v3.ts

```diff
@@ -250,7 +230,7 @@ router.post('/trade', async (req: Request, res: Response) => {
             );
 
       if (side === 'BUY') {
-        const price = trade.executionPrice.invert().toSignificant(8);
+        const price = parseFloat(trade.executionPrice.invert().toSignificant(8));
```

Comment:
Applying both `parseFloat` and `toSignificant` seems a bit complex. Are both needed here?